### PR TITLE
Changed user->the users, where it makes sense

### LINKED ISSUE #168: Users
We often used phrases like "The user ..., he ..."
As we decided to use the plural form, this has to be changed for the whole document.

### PR DESCRIPTION
**Pull request by RomanLangrehr**
Fixes #168

### CODE REVIEW

**Comment by m1cm1c** [comment]
Please fix conflicts.

**Comment by RomanLangrehr** [comment]
Done

**Comment by m1cm1c** [comment]
lgtm

File: doc/Requirements Specification/Parts/Discussion.lyx [hunk 1]
```diff
@@ -464,8 +464,8 @@ gls{component}
 \end_inset
 
  might be very effortful or even impossible.
- If this cannot be solved, Beagle may require the user to provide a test
- bed in which the measured components can be executed.
+ If this cannot be solved, Beagle may require users to provide a test bed
+ in which the measured components can be executed.
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Discussion.lyx]: `\gls` missing here!

File: doc/Requirements Specification/Parts/Functional Requirements.lyx [hunk 2]
```diff
@@ -698,7 +698,7 @@ name "/OF140/"
 
 \end_inset
 
- The user may resume a paused measurement (
+ Userss may resume a paused measurement (
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Functional Requirements.lyx]: “Users**[]**”

File: doc/Requirements Specification/Parts/Non-Functional Requirements.lyx [hunk 3]
```diff
@@ -904,7 +904,7 @@ gls{SoMoX}
 \end_layout
 
 \begin_layout Standard
-When the users launches the analysis, they are presented a window where
+When users launches the analysis, they are presented a window where
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Non-Functional Requirements.lyx]: “launch**[]**”

File: doc/Requirements Specification/Parts/Non-Functional Requirements.lyx [hunk 4]
```diff
@@ -929,7 +929,7 @@ reference "/OQ40/"
 
 \end_inset
 
- is implemented, the user may adapt the default timeout.
+ is implemented, the users may adapt the default timeout.
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Non-Functional Requirements.lyx]: “[…] implemented,**[]** users may […]”

File: doc/Requirements Specification/Parts/Non-Functional Requirements.lyx [hunk 5]
```diff
@@ -951,7 +951,7 @@ reference "/OQ40/"
 
 \end_inset
 
- is implemented, the user can select to additionally benchmark their hardware
+ is implemented, users can select to additionally benchmark their hardware
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Non-Functional Requirements.lyx]: “can select” -> “may choose” (I think that sounds nicer ;) )

File: doc/Requirements Specification/Parts/Purpose.lyx [hunk 6]
```diff
@@ -849,7 +849,7 @@ gls{Beagle}
 
 \end_inset
 
- provides the user to control the analysis with a 
+ provides users to control the analysis with a
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Purpose.lyx]: Am I getting tired or does this sentence make no sense at all?

How about:

“Beagle offers users a GUI to control the analysis”

**Comment by flyfl** [comment]
lgtm

File: doc/Requirements Specification/Parts/Purpose.lyx (hunk 6 above)
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Purpose.lyx]: By the way: Why is this optional?

Comment (flyfl) [inline doc/Requirements Specification/Parts/Purpose.lyx]: I think we can use your sentence.

**Comment by viridispirus** [comment]
lgtm

**Pull request merged**